edx_adapt/data/course_repository.py

```diff
@@ -396,6 +399,31 @@ def enroll_user(self, course_id, user_id):
         self.store.course_append(course_id, 'users_in_progress', user_id)
         self.store.record_data(coll, {'student_id': user_id, 'current': None, 'next': None})
 
+    def post_probabilities(self, course_id, prob_list, new=False):
+        """
+        Add default probability (or set of probabilities) student's skills are enrolled with.
+
+        :param course_id: course id
+        :param prob_list: list of dicts with probability parameters
+        :param new: boolean flag to mark that old probabilities is replaced by new
+        """
+        if isinstance(prob_list, list):
+            self.store.update_doc(
+                'Courses', {'course_id': course_id}, {('$set' if new else '$addToSet'): {'probabilities': prob_list}}
+            )
+        else:
+            logger.error("Probabilities are not given in a list: {}".format(prob_list))
+            raise interface.DataException("Incorrect type of the prob_list parameter")
+
+    def get_probabilities(self, course_id):
```

Comment:
Here and everywhere: probabilities -> model_params

Because probabilities is too broad